Makefile

```diff
@@ -21,49 +21,52 @@ test: $(CPDEPS) $(DPDEPS)
 
 VERSION                  ?= $(shell cat ./VERSION)
 DOCKER_TAG               ?= ${VERSION}
+DOCKER_DEBUG_TAG         ?= ${DOCKER_TAG}-debug
 DOCKER_REGISTRY          ?=
 DOCKER_REPOSITORY        ?=
-DOCKER_BUILD_ARGS        ?=
+DOCKER_BUILD_ARGS        ?= --build-arg RTE_MACHINE='hsw'
```

Comment:
@zdw @hyunsun should we add `RTE_MACHINE='hsw'` only in case of debug build? Is that the mcord-in-a-box image?

Reply:
The point of this pre-built image in Docker Hub is to provide a sample image that "just works" for a user who wants to try OMEC not just for comac-in-a-box - obviously, we cannot provide perfect image for all machines. Dockerfile and install script have `native` as its default machine type, and I don't think people even consider this pre-built image in a serious test or deployment. And we can add a note to deployment instructions.
If we remove this option here, machine type of the pre-built image in Dockerhub will change all the time because we use EC2 as our Jenkins build machine. Don't you think this is better than that?

Reply:
Couple of concerns - 

If you are promising some minimum then you cannot build on random machines in EC2
Then jenkinsfile can pass these extra args because its comac-in-a-box specific settings should not be default in this repo

Reply:
@krsna1729 It's not comac-in-a-box specific setting. As I mentioned, it will be our default image in OMEC helm charts(https://github.com/opencord/helm-charts/blob/master/mcord-release/mcord-services/values.yaml#L100).

Reply:
Lets document with a comment in the makefile as to why we are setting this

Reply:
Thanks for understanding. I'll add a note to Makefile.

Reply:
I added a note. Welcome corrections or additions if you have any.